File: DSA/Navneetpant_590011569/Day_10/Q2.java
class Solution{
    public String compareGender (String s) {
        if (s.length() % 2 == 0 ){
            return "CHAT WITH HER!";

        }
        else {
            return "IGNORE HIM";
        }
}
}
public class Q2 {
    public static void main(String[] args) {
        String ex1 = "sevenkplus";
        Solution temp = new Solution();
        System.out.println(temp.compareGender(ex1));
    }
    
}

File: DSA/Prarthana_Baisoya_590024838/Day_10/q2.c
#include <stdio.h>
#include <string.h>

int main()
{
    char s[100];
    scanf("%s", s);

    int freq[26] = {0};
    int count = 0;

    for(int i = 0; s[i] != '\0'; i++)
    {
        int pos = s[i] - 'a';

        if(freq[pos] == 0)
        {
            freq[pos] = 1;
            count++;
        }
    }

    if(count % 2 == 0)
    {
        printf("CHAT WITH HER!");
    }
    else
    {
        printf("IGNORE HIM!");
    }

    return 0;
}
